assets/queries/crossplane/azure/redis_cache_allows_non_ssl_connections/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "6c7cfec3-c686-4ed2-bf58-a1ec054b63fc",
+  "queryName": "Redis Cache Allows Non SSL Connections",
+  "severity": "MEDIUM",
+  "category": "Encryption",
```

Comment:
I think we should confirm with our security team, since the related queries does not have the same category